### DISCUSSION TITLE
Insert generated reply not working

### DISCUSSION THREAD

**Discussion by ccoch** [GorvGoyl/ChatGPT-Writer#12] [Report Bugs and Issues]
When I clic on the "Insert generated reply" button, it doesn't work, nothing happens. I use Google Chrome.

**Comment by Vmadmax** [comment]
It does not work for me in gmail either.

console.log:
```text
couldn't find gmail inputbox to insert text
```

**Comment by GorvGoyl** [comment]
fixed in latest release

**Comment by anewbies** [comment]
Se estiver usando o Google Chrome, entre na configuração chrome://settings/syncSetup e marque a caixa on "verificação de ortografia aprimorada" e o problema estará resolvido.
![image](https://user-images.githubusercontent.com/14927641/213434388-b31201f0-4ac8-4fcc-a0eb-626e83ff2ef3.png)
